.github/actions/set-up-composite/action.yml

```diff
@@ -0,0 +1,224 @@
+  steps:
+    - uses: actions/checkout@v3
+      with:
+        path: SUBMITTY_CPY/
+    - name: Copy Repo
+      run: |
+        # have to copy first, absolute paths not supported by actions/checkout@v3
+          sudo mkdir -p ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/Submitty
+          sudo cp -R SUBMITTY_CPY/. ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/Submitty
+          sudo chmod -R a+rwx  ${SUBMITTY_INSTALL_DIR}
+          sudo chmod -R a+rwx /tmp/
+      shell: bash
+    - uses: actions/setup-python@v4
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
+    - uses: shivammathur/setup-php@2.24.0
+      with:
+        php-version: ${{ env.PHP_VER }}
+        extensions: imagick, ldap
+
+    - name: Set Timezone
+      run: |
+        sudo timedatectl set-timezone America/New_York
+      shell: bash
+
+    - name: Cache pip
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-${{ github.job }}-pip-${{ hashFiles('**/system_requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ github.job }}-pip-
+
+    - name: Install Test-specific python dependencies
+      run: |
+        # Install for local
+        python3 -m pip install --upgrade pip setuptools wheel
+        pip3 -V
+        pip3 install selenium websocket_client
+        # Install globally
+        sudo python3 -m pip install --upgrade pip setuptools wheel
+        sudo pip3 -V
+      shell: bash
+    - name: Install python dependencies
+      run: |
+        # Install for local
+        pip3 install -r ${SUBMITTY_REPOSITORY}/.setup/pip/system_requirements.txt
+        # Install globally
+        sudo pip3 install -r ${SUBMITTY_REPOSITORY}/.setup/pip/system_requirements.txt
+      shell: bash
+    - name: Get composer cache dir
+      id: composer-cache
+      run: |
+        cd ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/Submitty/site
+        echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+      shell: bash
+    - name: Install composer Cache
+      uses: actions/cache@v3
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-php-composer-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-composer-
+
+    - name: Install PHP dependencies
+      run: |
+        cd  ${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/Submitty/site
+        composer install --prefer-dist
+      shell: bash
+
+    - name: Install Submitty python utils
+      run: |
+        cd ${SUBMITTY_REPOSITORY}/python_submitty_utils
+        pip3 install .
+        pip3 show submitty_utils
+        umask 022
+        sudo -H pip3 install .
+        sudo pip3 show submitty_utils
+      shell: bash
+
+    - name: Install accessibility checker
+      run: |
+        wget https://github.com/validator/validator/releases/download/20.3.16/vnu.jar_20.3.16.zip
+        unzip vnu.jar_20.3.16.zip
+        sudo mv dist/vnu.jar /usr/bin/
+      shell: bash
```

Comment:
I believe this only needs to be installed for the selenium tests (for the tests in `tests/e2e/test_accessibility.py` specifically)